test/test_images/sendevent/main.go

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
yes this is a good idea. I will add a flag to control this.

Reply:
@grantr when you say from a script, do you mean you prefer a separate pod being created, or ? Just clarifying so I understand your preference :)

Reply:
@vaikas-google I think it would be more convenient to make requests to eventing endpoints directly from the pod running the go test. I do not prefer a separate pod creation, though I don't have a strong preference either way. This seems like something that can be changed in a future PR.

Reply:
we could make a bounce pod, like just a simple proxy pod that gets configured with the sink and then exposes it's self for a moment while the test runs.

Reply:
That's my preference as well, I think creating new pods just to send an event seems unnecessary and makes it harder to debug test failures.

Reply:
Ok, so just to close the loop on this. I didn't realize that the reason for this is because tests run outside of the cluster and hence they don't have access to sending events directly to the exposed channels. My bad.